applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -36,6 +36,32 @@ bool le_audio_ep_state_check(struct bt_bap_ep *ep, enum bt_bap_ep_state state)
 	return false;
 }
 
+bool le_audio_ep_qos_configured(struct bt_bap_ep *ep)
+{
+	int ret;
+	struct bt_bap_ep_info ep_info;
+
+	if (ep == NULL) {
+		LOG_DBG("EP is NULL");
+		/* If an endpoint is NULL it is not in any of the states */
+		return false;
+	}
+
+	ret = bt_bap_ep_get_info(ep, &ep_info);
+	if (ret) {
+		LOG_WRN("Unable to get info for ep");
+		return false;
+	}
+
+	if (ep_info.state == BT_BAP_EP_STATE_IDLE ||
```

Comment:
This should look for positives and not negatives. I.e. it should check for valid states and return true. If another new state is added, it will return true for this possibly invalid state.